apps/angular/signal-input/src/app/user.component.ts

```diff
@@ -19,23 +20,20 @@ const ageToCategory = (age: number): Category => {
   standalone: true,
   imports: [TitleCasePipe],
   template: `
-    {{ fullName | titlecase }} plays tennis in the {{ category }} category!!
+    {{ fullName() | titlecase }} plays tennis in the {{ category() }} category!!
   `,
   host: {
     class: 'text-xl text-green-800',
   },
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class UserComponent implements OnChanges {
-  @Input({ required: true }) name!: string;
-  @Input() lastName?: string;
-  @Input() age?: string;
+export class UserComponent {
+  name = input.required();
+  lastName = input();
```

Comment:
I think you need to give a type otherwise you have lastName is of type `signal<unknown, unknown>`

Reply:
Thank you, I could check it. Fixed on next commit